charts/lagoon-build-deploy/Chart.yaml

```diff
@@ -16,11 +16,11 @@ kubeVersion: ">= 1.23.0-0"
 
 type: application
 
-version: 0.26.4
```

Comment:
Could this have been a patch version?

Reply:
possibly - but I always tend towards a minor release on an appversion bump, and reserve patches for minor functionality or fixups between?